client/me/developer/dev-survey-notice.tsx

```diff
@@ -0,0 +1,74 @@
+import { Gridicon } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import surveyImage from 'calypso/assets/images/illustrations/developer-survey.svg';
+
+type DevSurveyNotice = {
+	onClose: ( remindTimeInSeconds: number ) => void;
+	onOk: () => void;
```

Comment:
```suggestion
	onSurveyClick: () => void;
```
I would suggest a more specific name, at a glance, there's no way to know what `onOk` refers to, there is no "OK" button in this component.